plancton/__init__.py

```diff
@@ -210,18 +211,19 @@ def _read_conf(self):
       self.conf["max_docks"] = 0
     if not isinstance(self.conf["docker_cmd"], list):
       self.conf["docker_cmd"] = self.conf["docker_cmd"].split(" ")
-    if self.conf["influxdb_url"] and not "#" in self.conf["influxdb_url"]:
-      # Invalid InfluxDB URL: disable monitoring
-      self.conf["influxdb_url"] = None
+    tmp_url_list = []
```

Comment:
So, I like it because if param is a string it is transformed to a list; if it's a list then you retain only what's OK; if it's none of the above then it is silently ignored.

A note. On Python 2 you would check `isinstance(..., basestring)` instead of `str` in order to catch the unicode case too. But I would leave it like this as no one uses unicode URLs, and in any case we don't have a smooth way to make it work on Python 3 too.

The whole code can be written without using the temporary variable, which makes it clearer (if I am not missing something):

```
    if isinstance(self.conf["influxdb_url"], str):
      self.conf["influxdb_url"] = [ self.conf["influxdb_url"] ]
    if isinstance(self.conf["influxdb_url"], list):
      self.conf["influxdb_url"] = [x for x in self.conf["influxdb_url"] and "#" in x]
    else:
      self.conf["influxdb_url"] = []
```